crates/uv/tests/it/tool_install.rs

```diff
@@ -1974,7 +1974,7 @@ fn tool_install_requirements_txt_arguments() {
     ----- stdout -----
 
     ----- stderr -----
-    Installed 2 executables: black, blackd
+    `black` is already installed
```

Comment:
As an example: we didn't detect that this install request was identical to the previous, because in this latter case, the requirement came from a `requirements.txt` file rather than the CLI.